code/drasil-example/projectile/lib/Drasil/Projectile/Concepts.hs

```diff
@@ -31,15 +31,16 @@ projMotion = compoundNC projectile motion
 ---
 
 defs :: [ConceptChunk]
-defs = [launcher, projectile, target, projSpeed]
+defs = [launcher, projectile, target, projSpeed, projPos]
 
-launcher, projectile, target, projSpeed :: ConceptChunk
+launcher, projectile, target, projSpeed, projPos :: ConceptChunk
 launcher   = dcc "launcher"   (nounPhraseSP "launcher")  ("where the projectile is launched from " ++
                                                           "and the device that does the launching")
 projectile = dcc "projectile" (nounPhraseSP "projectile") "the object to be launched at the target"
 target     = dcc "target"     (nounPhraseSP "target")     "where the projectile should be launched to"
 
 projSpeed  = dccWDS "projSpeed" (nounPhraseSP "1D speed") (S "1D speed under" +:+ phrase constant +:+ phrase acceleration)
+projPos  = dccWDS "projPos" (nounPhraseSP "1D position") (S "1D position under" +:+ phrase constant +:+ phrase speed)
```

Comment:
For "database-like" entries like this, try to "align" the arguments/operators as best as possible (e.g., `projectile` and `target`).

Reply:
Also, `getAcc oneD` should be used instead of `S "1D"` to make use of our concepts (including in `projSpeed`, which I know isn't part of this PR).

Reply:
Ok thanks! I will make those changes.